recipes/datalad/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "datalad" %}
+{% set version = "0.9.3" %}
+{% set sha256 = "081a4dba4a046a949744dd94232c88fe22fca4decb4ff77edbac842183824944" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - datalad=datalad.cmdline.main:main
+    - git-annex-remote-datalad-archives=datalad.customremotes.archives:main
+    - git-annex-remote-datalad=datalad.customremotes.datalad:main
+
+requirements:
+  build:
+    - appdirs
+    - boto
+    - gitpython >=2.1.8
+    - humanize
+    - iso8601
+    - jsmin
+    - keyring >=8.0
+    - keyrings.alt
+    - mock
+    - msgpack-python
+    - patool >=1.7
+    - pyld
+    - python
+    - requests >=1.2
+    - setuptools
+    - simplejson
+    - six >=1.8.0
+    - tqdm
```

Comment:
This seems like a lot of build reqs, are all of them needed?

Reply:
indeed, should not be needed I think. Thanks, will tune up for that now too